### PR TITLE
Testing course reg stu list

### PR DESCRIPTION
**Pull request by Michelle-Lohwt**
1. Comment unnecessary imports or code
2. Unblock buttons
3. Fix indentation
4. Fix course description and course time scrolling UI problem

Problems
1. stuList search function for courses not functioning (solved)
2. TableView not complete
3. stuList search function for students not complete

https://user-images.githubusercontent.com/60863955/150663471-49dc56bf-e95b-45f0-997c-6a54f02bb10a.mp4

### CODE REVIEW

File: src/sample/courseRegController.java [hunk 1]
```diff
@@ -128,101 +132,101 @@ public void addcourse() {
...
       updatebothlist();
 
-      //Clear the Course Details after a course is registered.
+      // Clear the Course Details after a course is registered.
       courseTitle.setText("");
       time.setText("");
       desc.setText("");
-
-      addCourseButton.setDisable(true);	
-      removeCourseButton.setDisable(true);
```
Comment (GitHubUser5136) [inline src/sample/courseRegController.java:225]: May I know why the `setDisable(true)` for both of these buttons is removed?

File: src/sample/courseRegController.java [hunk 2]
```diff
@@ -128,101 +132,101 @@ public void addcourse() {
...
-        
-      //Clear the Course Details after a course is registered.
+
+      // Clear the Course Details after a course is registered.
       courseTitle.setText("");
       time.setText("");
       desc.setText("");
-
-      addCourseButton.setDisable(true);	
-      removeCourseButton.setDisable(true);
```
Comment (GitHubUser5136) [inline src/sample/courseRegController.java:155]: May I know why the setDisable(true) for both of these buttons is removed?

File: src/sample/courseRegController.java [hunk 3]
```diff
@@ -335,48 +339,48 @@ public void updatebothlist() {
     displaycourselist();
   }
 
-  //When a Course in the Course List is Clicked, this function will be executed.	
-  public void courselistdetails() {	
-    if(courseList.getSelectionModel().getSelectedItem() != null){	
-      	
-      addCourseButton.setDisable(false);
```
Comment (GitHubUser5136) [inline src/sample/courseRegController.java:342]: May I know why the `setDisable(false)` for this button is removed?

File: src/sample/stuListController.java [hunk 4]
```diff
@@ -148,9 +151,6 @@ public void addcourse(){
     courseTitle.setText("");
     time.clear();
     desc.clear();
-
-    addCourseButton.setDisable(true);	
-    removeCourseButton.setDisable(true);
```
Comment (GitHubUser5136) [inline src/sample/stuListController.java:153]: May I know why the `setDisable(true)` for these buttons are removed?

File: src/sample/stuListController.java [hunk 5]
```diff
@@ -194,9 +194,6 @@ public void removecourse(){
...
```
Comment (GitHubUser5136) [inline src/sample/stuListController.java:199]: May I know why the `setDisable(true)` for these buttons are removed?

File: src/sample/stuListController.java [hunk 6]
```diff
@@ -467,9 +464,6 @@ public void searchstudent(){
   public void initialize(URL location, ResourceBundle resources) {
     displaycourselist();
     displayteachingcourse();
-    addCourseButton.setDisable(true);
-    removeCourseButton.setDisable(true);
-    editCourseButton.setDisable(true);
```
Comment (GitHubUser5136) [inline src/sample/stuListController.java:472]: May I know why the `setDisable(true)` for these buttons are removed?

File: src/sample/courseRegController.java (hunk 1 above)
Comment (Michelle-Lohwt) [reply src/sample/courseRegController.java:225]: If you disable, how can the users add course or remove course after this function runs?

File: src/sample/courseRegController.java (hunk 2 above)
Comment (Michelle-Lohwt) [reply src/sample/courseRegController.java:155]: Same as above comment

File: src/sample/courseRegController.java (hunk 3 above)
Comment (Michelle-Lohwt) [reply src/sample/courseRegController.java:342]: This one can be deleted since this is extra

File: src/sample/stuListController.java (hunk 4 above)
Comment (Michelle-Lohwt) [reply src/sample/stuListController.java:153]: Same as above

File: src/sample/stuListController.java (hunk 5 above)
Comment (Michelle-Lohwt) [reply src/sample/stuListController.java:199]: Same as above

File: src/sample/stuListController.java (hunk 6 above)
Comment (Michelle-Lohwt) [reply src/sample/stuListController.java:472]: Users are not able to do anything since you disable all the buttons

**Pull request merged**